### PR TITLE
Refresh download page

### PR DESCRIPTION
**Pull request by slawekjaranowski**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MEAR) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[MEAR-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MEAR-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by slawekjaranowski** [comment]
Now looks as

image

File: src/site/xdoc/download.xml.vm [hunk 1]
```diff
@@ -23,77 +23,26 @@ under the License.
...
-            [end]
-            [if-any backup]
-            [for backup]
-            <option value="[backup]">[backup] (backup)</option>
-            [end]
-            [end]
-          </select>
-          <input type="submit" value="Change"/>
-        </form>
+      <p>In order to guard against corrupted downloads/installations, it is highly recommended to
```
Comment (kwin) [inline src/site/xdoc/download.xml.vm:36]: The signature checking is for verifying whether the downloaded artifact has been created by a trusted person. Corrupted downloads are detected by checksums.

Comment (michael-o) [reply src/site/xdoc/download.xml.vm:36]: Correct, both serve different purposes.

**Review by michael-o — CHANGES_REQUESTED** [review]

File: src/site/xdoc/download.xml.vm (hunk 1 above)
Comment (slawekjaranowski) [reply src/site/xdoc/download.xml.vm:36]: What is proposition for such sentence ...
The same we have on the Maven download page 😄

**Comment by slawekjaranowski** [comment]
Next proposition:

image

Inspired by ASF download template: https://www.apache.org/dyn/closer.lua

**Comment by kwin** [comment]
Still not convinced TBH ;-). Integrity is usually checked by Maven Resolver, only authenticity needs to be verified manually.

**Comment by michael-o** [comment]
> Still not convinced TBH ;-). Integrity is usually checked by Maven Resolver, only authenticity needs to be verified manually.

I agree here. There is a clear distinction between checksum and a signature.

**Comment by slawekjaranowski** [comment]
It is download page where we put link to source release of project from ASF distribution area - it is not binary version of plugin which is downloaded by Maven.

Users download it manually and we give instruction how to verify it.

**Comment by michael-o** [comment]
> It is download page where we put link to source release of project from ASF distribution area - it is not binary version of plugin which is downloaded by Maven.
> 
> Users download it manually and we give instruction how to verify it.

If you, we should require to the check the signature only, not the checksum since sig check will include checksum.

**Comment by michael-o** [comment]
Me and @cstamas have expressed many times that this ASF page requires a cleanup because it mixes hashes and checksums throughout.

**Comment by slawekjaranowski** [comment]
ok, my goal was a refresh a little download page, remove broken links to mirrors 

changing ASF release policy and related documents are out of scope for this PR 😄

File: src/site/xdoc/download.xml.vm [hunk 2]
```diff
@@ -23,102 +23,51 @@ under the License.
...
+              <td>${project.name} ${project.version} (Source zip)</td>
+              <td><a href="[preferred]maven/plugins/${project.artifactId}-${project.version}-source-release.zip">${project.artifactId}-${project.version}-source-release.zip</a></td>
+              <td><a href="https://downloads.apache.org/maven/plugins/${project.artifactId}-${project.version}-source-release.zip.sha512">${project.artifactId}-${project.version}-source-release.zip.sha512</a></td>
+              <td><a href="https://downloads.apache.org//maven/plugins/${project.artifactId}-${project.version}-source-release.zip.asc">${project.artifactId}-${project.version}-source-release.zip.asc</a></td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p>It is essential that you <a href="https://www.apache.org/info/verification.html">verify the integrity</a> of the downloaded file
+          using a hash (.sha512 file)
```
Comment (michael-o) [inline src/site/xdoc/download.xml.vm:62]: the checksum

**Review by michael-o — CHANGES_REQUESTED** [review]
Last nit and then we can merge

**Comment by michael-o** [comment]
> ok, my goal was a refresh a little download page, remove broken links to mirrors
> 
> changing ASF release policy and related documents are out of scope for this PR 😄

I know and I appreciate that. It is a different discussion which has been bugigng me for quite some time.

**Review by cstamas — APPROVED** [review]

**Pull request merged**